kubernetes/resource_kubernetes_persistent_volume_test.go

```diff
@@ -1834,10 +1834,6 @@ func testAccKubernetesPersistentVolumeConfig_hostPath_nodeAffinity_match(name, s
 	return testAccKubernetesPersistentVolumeConfig_hostPath_nodeAffinity(name, nodeAffinity)
 }
 
-func testAccKubernetesPersistentVolumeConfig_hostPath_withoutNodeAffinity(name string) string {
```

Comment:
Hi @thevilledev,

All looks good, but could you please comment put this function instead of removing it? It is used in one of the commented steps above and we could come back to this issue at some point.

Thank you!

Reply:
Hey @arybolovlev, thanks for pointing this out! I somehow totally missed it while `grep`'ing potential (commented out) references. Commented it out now